src/components/access_group/AccessGroupMembersEdit.vue

```diff
@@ -0,0 +1,734 @@
+<template>
+  <section class="edit-members-container">
+    <AccessGroupEditPanel
+      :title="`${fluent('access-group_members')} (${allMembers.length})`"
+      :full="true"
+    >
+      <template v-slot:content>
+        <div class="members-list-container">
+          <SearchForm
+            class="edit-members__search"
+            v-on:clear-query="clearSearchHandler"
+            :searchFormHandler="searchFormHandler"
+            :searchFormLabel="
+              fluent('access-group_members', 'edit-members__search')
+            "
+          ></SearchForm>
+          <table class="edit-members__table">
```

Comment:
This could be a nice use case for grid + table, or maby flexbox. However, not duplicating lists would be preferable I guess.